python-checks/src/test/java/org/sonar/python/checks/ClassMethodFirstArgumentNameCheckTest.java

```diff
@@ -19,13 +19,39 @@
  */
 package org.sonar.python.checks;
 
+import java.util.List;
 import org.junit.Test;
+import org.sonar.plugins.python.api.PythonCheck;
 import org.sonar.python.checks.utils.PythonCheckVerifier;
+import org.sonar.python.checks.utils.PythonQuickFixVerifier;
+import org.sonar.python.quickfix.IssueWithQuickFix;
+
+import static org.assertj.core.api.Assertions.assertThat;
 
 public class ClassMethodFirstArgumentNameCheckTest {
 
   @Test
   public void test() {
     PythonCheckVerifier.verify("src/test/resources/checks/classMethodFirstArgumentNameCheck.py", new ClassMethodFirstArgumentNameCheck());
+
+    String codeWithIssue = "class A():\n\n" +
+      "    @classmethod\n" +
+      "    def area(bob, length, width):\n" +
+      "        return length*width\n";
```

Comment:
I think the code example could be simplified: fewer parameters, trivial body (`pass`)...